From dc93b33e52559d75e0edd08c8e8151cf07a1e438 Mon Sep 17 00:00:00 2001
From: Marcin Wojtas <mw@semihalf.com>
Date: Wed, 24 Aug 2016 19:10:07 +0200
Subject: [PATCH 0914/2241] arm64: marvell: enable CP110 syscon driver

This patch is part of patch-set which adds
mainline mvebu clock gating support [9/21].

Change-Id: I2ed266d5fd7b3d30c1a61ca6f25b68e5909568eb
Signed-off-by: Marcin Wojtas <mw@semihalf.com>
Signed-off-by: Igal Liberman <igall@marvell.com>
Reviewed-on: http://vgitil04.il.marvell.com:8080/32394
Reviewed-by: Omri Itach <omrii@marvell.com>
Tested-by: Omri Itach <omrii@marvell.com>
---
 arch/arm64/Kconfig.platforms | 1 +
 1 file changed, 1 insertion(+)

diff --git a/arch/arm64/Kconfig.platforms b/arch/arm64/Kconfig.platforms
index 372cba3..bec12af 100644
--- a/arch/arm64/Kconfig.platforms
+++ b/arch/arm64/Kconfig.platforms
@@ -12,6 +12,7 @@ config ARCH_MVEBU
 	select ARMADA_AP806_CORE_CLK
 	select ARMADA_AP806_RING_CLK
 	select ARMADA_AP806_SYSCON
+	select ARMADA_CP110_SYSCON
 	select PINCTRL_ARMADA_AP806
 	select PINCTRL_ARMADA_CP110
 	help
-- 
2.7.4

